app/assets/stylesheets/layout/_profile.scss

```diff
@@ -1,122 +1,128 @@
-@import "bootstrap";
+// @import "bootstrap";
 @import "../base/*";
 
 .profile {
-	padding-top: 35px;
-	background: #e5e3d3;
-	label {
-		font-size: 20px;
-		font-weight: normal;
-		color: #6c6c6c;
-	}
-	.form-control {
-		padding: 20px 26px;
-		font-size: 14px;
-		height: auto;
-		color: #7e7e7d;
-		background-color: #f4f4f1;
-		background-image: none;
-		border: none;
-		border-radius: 0;
-		box-shadow: none;
-		transition: none;
-	}
-	.form-control:hover {
-		box-shadow: none;
-		transition: none;
-	}
+  padding-top: 35px;
+  background: #e5e3d3;
+  label {
+    font-size: 20px;
+    font-weight: normal;
+    color: #6c6c6c;
+  }
+  .form-control {
+    padding: 20px 26px;
+    font-size: 14px;
+    height: auto;
+    color: #7e7e7d;
+    background-color: #f4f4f1;
+    background-image: none;
+    border: none;
+    border-radius: 0;
+    box-shadow: none;
+    transition: none;
+  }
+  .form-control:hover {
+    box-shadow: none;
+    transition: none;
+  }
 }
+
 .panel-heading {
-	padding: 0;
-	border: none;
-	margin: 0;
-	h3 {
-		background: #efeddf;
-		margin: 0;
-		padding: 22px 20px;
-		font-size: 20px;
-		color: #952c2c;
-		font-weight: bold;
-	}
+  padding: 0;
+  border: none;
+  margin: 0;
+  h3 {
+    background: #efeddf;
```

Comment:
Color literals like `#efeddf` should only be used in variable declarations; they should be referred to via variable everywhere else.